[2022-06-16 00:00:13,360] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: say_hi.get_teams_task scheduled__2022-06-15T00:00:00+00:00 [queued]>
[2022-06-16 00:00:13,433] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: say_hi.get_teams_task scheduled__2022-06-15T00:00:00+00:00 [queued]>
[2022-06-16 00:00:13,434] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 00:00:13,434] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-16 00:00:13,434] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 00:00:13,539] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): get_teams_task> on 2022-06-15 00:00:00+00:00
[2022-06-16 00:00:13,547] {standard_task_runner.py:52} INFO - Started process 2314 to run task
[2022-06-16 00:00:13,569] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'say_hi', 'get_teams_task', 'scheduled__2022-06-15T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/username.py', '--cfg-path', '/tmp/tmp8a2hczw2', '--error-file', '/tmp/tmp_rm7q1cp']
[2022-06-16 00:00:13,570] {standard_task_runner.py:80} INFO - Job 30: Subtask get_teams_task
[2022-06-16 00:00:13,797] {task_command.py:370} INFO - Running <TaskInstance: say_hi.get_teams_task scheduled__2022-06-15T00:00:00+00:00 [running]> on host 6a14fd44e8dc
[2022-06-16 00:00:14,231] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=say_hi
AIRFLOW_CTX_TASK_ID=get_teams_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-15T00:00:00+00:00
[2022-06-16 00:00:14,233] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-16 00:00:14,234] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "getting three random teams"']
[2022-06-16 00:00:14,265] {subprocess.py:85} INFO - Output:
[2022-06-16 00:00:14,269] {subprocess.py:92} INFO - getting three random teams
[2022-06-16 00:00:14,270] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-16 00:00:14,442] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=say_hi, task_id=get_teams_task, execution_date=20220615T000000, start_date=20220616T000013, end_date=20220616T000014
[2022-06-16 00:00:14,551] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-16 00:00:14,714] {local_task_job.py:273} INFO - 3 downstream tasks scheduled from follow-on schedule check
